runtime/gc_realtime/RealtimeAccessBarrier.cpp

```diff
@@ -950,5 +950,15 @@ MM_RealtimeAccessBarrier::forwardReferenceArrayCopyIndex(J9VMThread *vmThread, J
 	return -2;
 }
 
+bool
+MM_RealtimeAccessBarrier::preMountContinuation(J9VMThread *vmThread, j9object_t contObject)
+{
+	if (_realtimeGC->isCollectorConcurrentTracing()) {
```

Comment:
don't know if it's the same, but use isBarrierActive as it's already case in the rest of this file